tensorflow/c/c_api.cc

```diff
@@ -1667,6 +1667,25 @@ void TF_DeleteGraph(TF_Graph* g) {
   if (del) delete g;
 }
 
+size_t TF_GraphOperationsCount(TF_Graph* graph) {
```

Comment:
Ya - sure, it's not a grave thing; i just wanted to make sure i understood the salient actors of the method before i went ahead with the implementation.
Ok - thanks for the sanity check.